extjsdk/src/test/java/io/vantiq/extjsdk/TestExtensionWebSocketListener.java

```diff
@@ -115,7 +116,7 @@ public void testSourceConnection() {
     public void testSourceConnectionFailFirst() {
         authenticate(true);
         
-        listener.onMessage(TestListener.errorMessage());
+        listener.onMessage(null, TestListener.errorMessage());
```

Comment:
May have to extend the client or listener to return the `FalseWebSocket` they use to pass in the right thing here.